django/contrib/admin/static/admin/js/related-widget-wrapper.js

```diff
@@ -0,0 +1,21 @@
+django.jQuery(function($){
+    function updateLinks() {
+        var $this = $(this);
+        var siblings = $this.nextAll('.change-related, .delete-related');
+        if (!siblings.length) return;
+        var value = $this.val();
+        if (value) {
+            siblings.each(function(){
+                var elm = $(this);
+                elm.attr('href', elm.attr('data-href-template').replace('__pk__', value));
+            });
+        } else siblings.removeAttr('href');
+    }
+    var container = $('#container');
```

Comment:
in one of my custom admins, I don't have an id="container". Is `$('#container')` better than `$(document)`?